CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+
+- Building and maintaining workshops that teach about the protocols underlying the decentralized web, hosted at [https://proto.school/#/](https://proto.school/#/)
+- Organizing educational events around the world
+
+As a community, we support the following principles:
+
+- Beginners should feel welcome
+- Chapters should experiment with new ways to engage their community, and share approaches with other chapters over time
+- Events should be affordable and should strive towards diversity and inclusiveness
+- We are 100% open source and like to license things as open source/creative commons
+
+### Where does the name come from?
+
+The "Proto" in ProtoSchool represents the networking protocols that support the decentralized web, and is a shout-out to [Protocol Labs](https://protocol.ai/), the organization that sponsors our work.
+
+We also take a _lot_ of inspiration from [NodeSchool](https://nodeschool.io/). (Imitation is the sincerest form of flattery, and all that jazz.)
+
+### Ways to contribute
+
+#### Join (or start) a local chapter
+
+See if there is a [chapter near you](http://proto.school/chapters.html). Each chapter gets their own github repo, e.g. https://github.com/protoschool/san-francisco. These repos usually serve at least two purposes (maybe more):
+
+- host a chapter page on the `gh-pages` branch ([example](https://github.com/protoschool/san-francisco/tree/gh-pages))
+- provice a discussion board for the chapter using GitHub Issues ([example](https://github.com/protoschool/san-francisco/issues))
+
+There might be an issue for new or prospective members to introduce themselves, or there may be one that is an announcement/planning for the next event. To receive emails when new discussions happen, be sure to click "Watch" on your chapter's repo.
+
+If there is no local chapter near you and you'd like to start one, [check out our guide](https://github.com/protoschool/organizing/#how-to-start-a-new-protoschool-chapter) for instructions on how to get started.
+
+#### Help answer questions and provide tech support
+All of our workshops point people at this issue board when they get stuck:
+
+[https://github.com/protoschool/protoschool.github.io/issues](https://github.com/ipfs-shipyard/proto.school/issues)
+
+Questions we receive there range from beginner to advanced. You can click "Watch" on that repo to get emails in your inbox when new people ask questions. This helps us out a lot!
+
+In some cases, the best way to help will be by answering questions directly. In others, it will be to improve the workshop so the instructions are more clear (see below).
+
+#### Improve a workshop
+
+The code for our workshops lives at https://github.com/protoschool/protoschool.github.io and is maintained by @terichadbourne, with lots of help from our amazing volunteers. You can check out the [open issues](https://github.com/protoschool/protoschool.github.io/issues/?is=open) to find one you can jump in and help with.
```

Comment:
Let's put a list of active maintainers here, even if it's just you at first we can make it clearer that other contributors can become maintainers as well.

Reply:
Should I start with you and me, or are there other folks I should include for now?

Reply:
Put the two of us in for now. Most people I know are scared off by being labelled a "maintainer," so I haven't added the other folks who've been helping to date.